Readability.js

```diff
@@ -1678,15 +1678,18 @@ Readability.prototype = {
    *
    * @return boolean Whether or not we suspect parse() will suceeed at returning an article object.
    */
-  isProbablyReaderable: function() {
-    var nodes = this._doc.getElementsByTagName("p");
-    if (nodes.length < 5) {
-      return false;
-    }
-
-    var possibleParagraphs = 0;
+  isProbablyReaderable: function(helperIsVisible) {
```

Comment:
That's somewhat surprising to me... if you ask a child/descendant for display/visibility and its parent/ancestor is display: none, the child won't have that style (but obviously still won't be visible).